Header header
bool motors_state
bool frontBumpers
bool rearBumpers
float64 battery_state
float64 posX
float64 posY
float64 posW
